src/GraphQL/Instrumentation/ApolloTrace.cs

```diff
@@ -1,11 +1,8 @@
 using System;
 using System.Collections.Generic;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Serialization;
 
 namespace GraphQL.Instrumentation
 {
-    [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
```

Comment:
Contract for `ApolloTrace` should be configured in the custom contract resolver along with contract for `ExecutionResult`.

Reply:
Yep this should be done now.